views/shop/index.hbs

```diff
@@ -1,3 +1,17 @@
+
+<p>
```

Comment:
Why p tag is here

Reply:
because its simple tag to force new paragraph ...if you know another tag to use then use it

Reply:
fixed